fluent-package/yum/systemd-test/update-to-next-version-with-active-and-disabled.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/commonvar.sh
+
+# Install the current
+package="/host/${distribution}/${DISTRIBUTION_VERSION}/x86_64/Packages/fluent-package-[0-9]*.rpm"
+sudo $DNF install -y $package
+
+# The service should NOT start automatically
+(! systemctl is-active fluentd)
+# The service should be DISabled by default
+(! systemctl is-enabled fluentd)
+
+# Start the service
+sudo systemctl start fluentd
+systemctl is-active fluentd
+
+main_pid=$(eval $(systemctl show fluentd --property=MainPID) && echo $MainPID)
+
+# Make a dummy pacakge for the next version
+case $distribution in
+    amazon)
+        case $version in
+	    2023)
+		curl -L -o rpmrebuild.noarch.rpm https://sourceforge.net/projects/rpmrebuild/files/latest/download
+		sudo $DNF install -y ./rpmrebuild.noarch.rpm
+		;;
```

Comment:
could not get it from epel-release?

Reply:
@kenhys 
Wasn't there some reason why we couldn't use epel with Amazon Linux 2023?

* https://github.com/fluent/fluent-package-builder/pull/603

Reply:
Seems that AmazonLinux 2023 does not support EPEL.

> Amazon Linux 2023 (AL2023) doesn't support EPEL. For more information, see [Extra Packages for Enterprise Linux (EPEL)](https://docs.aws.amazon.com/linux/al2023/ug/compare-with-al2.html#epel).

https://repost.aws/knowledge-center/ec2-enable-epel

Reply:
Looks good as is.